packages/synapse-interface/components/StateManagedBridge/BridgeExchangeRateInfo.tsx

```diff
@@ -26,7 +26,7 @@ const BridgeExchangeRateInfo = () => {
       <section className="p-2 space-y-1 text-sm border rounded-sm border-[#504952] text-secondary font-light">
         <GasDropLabel />
         <Router />
-        <Fee />
+        {/* <Fee /> */}
```

Comment:
Let's just delete it altogether along with `Fee` definition below

Reply:
Sounds good!

Reply:
Removed in https://github.com/synapsecns/sanguine/commit/c36f78bf2d21937c76eff39a68465f65af0a4c02